protocols/gossipsub/src/peer_score.rs

```diff
@@ -338,6 +341,12 @@ impl PeerScore {
             let excess = peer_stats.behaviour_penalty - self.params.behaviour_penalty_threshold;
             let p7 = excess * excess;
             score += p7 * self.params.behaviour_penalty_weight;
+
+            // Slow peer weighting
+            if peer_stats.slow_peer_penalty > self.params.slow_peer_threshold {
+                let excess = peer_stats.slow_peer_penalty - self.params.slow_peer_threshold;
+                score += excess * self.params.slow_peer_weight;
+            }
```

Comment:
We should move this nested `if` statement (`Slow peer weighting`) outside of `P7: behavioural pattern penalty`?

https://github.com/sigp/lighthouse/blob/bbe9242811b5dc14d991506bc4d076f0ce98b3cb/beacon_node/lighthouse_network/gossipsub/src/peer_score.rs#L339-L350